glossary.yml

```
@@ -628,6 +628,11 @@
       Αλγόριθμος είναι ένα σύνολο από βήματα ή κανόνες για τη διεκπεραίωση μιας συγκεκριμένης εργασίας.
       Στην πληροφορική, αλγόριθμος είναι ένα σύνολο από εντολές σε κάποιο πρόγραμμα οι οποίες επιλύουν κάποιο
       υπολογιστικό πρόβλημα.
+  it:
+    term: "algoritmo"
+    def: >
+      Un algoritmo è una sequenza finita di regole ed operazioni da eseguire al fine di realizzare un certo compito.
+      In informatica, un algoritmo è un insieme di istruzioni presenti in un programma che permette di risolvere un problema computazionale.
```

```suggestion
      In informatica, un algoritmo è una sequenza finita di istruzioni presenti in un programma che permette di risolvere un problema computazionale.
```

Un insieme è una collezione non ordinata, mentre in un algoritmo è importante che le instruzioni vengano eseguite nell'ordine specificato. Credo "sequenza ordinata" descriva meglio la definizione.